music21/musicxml/xmlToM21.py

```diff
@@ -1716,16 +1724,16 @@ def separateOneStaff(streamPartStaff: stream.PartStaff, staffNumber: int):
                         copyVoice.insert(elem.offset, elem)
```

Comment:
Because this only walks measures and voices in the source, this is why spanners don't get propagated back into the templates with all the other notes/clefs etc. (spanner storage is one or two levels up.) I can add a comment if you like.